packages/livechat/src/routes/Chat/component.js

```diff
@@ -217,19 +237,26 @@ class Chat extends Component {
 								this.notifyEmojiSelect = click;
 							}}
 							handleEmojiClick={this.handleEmojiClick}
+							isRecording={this.state.isRecording}
+							handleRecording={this.handleRecording}
 							pre={
 								<ComposerActions>
-									<ComposerAction className={createClassName(styles, 'emoji-picker-icon')} onClick={this.toggleEmojiPickerState}>
+									<ComposerAction isDisabled={this.state.isRecording !== false} onClick={this.toggleEmojiPickerState}>
 										<EmojiIcon width={20} height={20} />
 									</ComposerAction>
 								</ComposerActions>
 							}
 							post={
 								<ComposerActions>
 									{text.length === 0 && uploads && (
-										<ComposerAction onClick={this.handleUploadClick}>
-											<PlusIcon width={20} height={20} />
-										</ComposerAction>
+										<ComposerActions>
+											<ComposerAction onClick={this.handleRecording}>
+												<MicIcon width={20} height={20} />
+											</ComposerAction>
+											<ComposerAction onClick={this.handleUploadClick} isDisabled={this.state.isRecording !== false}>
```

Comment:
```suggestion
											<ComposerAction onClick={this.handleUploadClick} isDisabled={this.state.isRecording}>
```
Wouldn't this be the same?